### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by Domino987**
## Related Issue
Relate the Github issue with this PR using `#`

## Description
Simple words to describe the overall goals of the pull request's commits.

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Impacted Areas in Application
List general components of the application that this PR will affect:

*

## Additional Notes
This is optional, feel free to follow your hearth and write here :)

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -33,8 +33,6 @@ export interface Action {
 export interface Column {
   cellStyle?: React.CSSProperties | ((data: any) => React.CSSProperties);
   currencySetting?:{ locale?: string,currencyCode?: string,minimumFractionDigits?:number,maximumFractionDigits?:number};
-  customFilterAndSearch?: (filter: any, rowData: any, columnDef: Column) => boolean;
```
Comment (mbrn) [inline index.d.ts]: You should not remove these lines. Please pull from material-table to your own fork.

File: index.d.ts [hunk 2]
```diff
@@ -34,7 +34,6 @@ export interface Column {
   cellStyle?: React.CSSProperties | ((data: any) => React.CSSProperties);
   currencySetting?:{ locale?: string,currencyCode?: string,minimumFractionDigits?:number,maximumFractionDigits?:number};
   customFilterAndSearch?: (filter: any, rowData: any, columnDef: Column) => boolean;
-  customSort?: (rowData1: any, rowData2: any) => number;
```
Comment (mbrn) [inline index.d.ts]: This should not be removed.

File: index.d.ts [hunk 3]
```diff
@@ -23,7 +23,8 @@ export interface MaterialTableProps {
 }
 
 export interface Action {
-  icon: string | React.ReactElement<any>;
+  customSort?: (rowData1: any, rowData2: any) => number;
```
Comment (mbrn) [inline index.d.ts]: This is wrong place of customSort

File: index.d.ts (hunk 1 above)
Comment (Domino987) [reply index.d.ts]: Sorry , failed to do that, fixed it

**Pull request closed without merging**